backend/routes/content.py

```
@@ -104,41 +104,82 @@ def calculate_completion_percentage(total_modules, completed_modules):
+    unit_id = module.unit_id
+    module_type = module.module_type
+    order = module.order
 
-    # Serve the HTML file
-    return send_file(file_path, mimetype="text/html")
+    base_content_dir = "content"
 
+    # Determine the file paths based on module type
+    if module_type == ModuleType.CONCEPT_GUIDE:
+        file_path = os.path.join(
+            base_content_dir, f"unit_{unit_id}", f"{order}_concept_guide.html"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)